src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -82,3 +110,10 @@
            :description     (i18n/label :t/empty-tab-description)
            :placeholder?    true
            :container-style style/empty-container-style}])])))
+
+(defn view
+  []
+  (reagent/create-class
+   (let [accounts (rf/sub [:profile/wallet-accounts])]
+     {:component-did-mount #(rf/dispatch [:wallet-2/get-wallet-tokens accounts])
+      :reagent-render      reagent-render})))
```

Comment:
```suggestion
(defn view
  []
  (rf/dispatch [:wallet-2/get-wallet-tokens])
  (fn []
     [reagent-render]})))
```

this form is simpler, you can just dispatch event on component initialization outside the render function, and you don't need extra reagent-render component, you can move it here in the view, also  you need to take account from db in the `:wallet-2/get-wallet-tokens`

Reply:
Done